neat_cache/lib/src/providers/inmemory.dart

```diff
@@ -104,7 +104,7 @@ class InMemoryCacheProvider<T> extends CacheProvider<T> {
   @override
   Future<void> close() async {
     _isClosed = true;
-    _old = null;
-    _new = null;
+    _old = {};
```

Comment:
nit: if the `_old` and `_new` fields are final, this could be `_old.clear()` instead. not sure if it makes any difference...

Reply:
If they are final you can't swap them...